pypose/lietensor/utils.py

```diff
@@ -266,139 +267,186 @@ def randn_like(input, sigma=1, **kwargs):
-            Default: if None, uses the current device for the default tensor
-            type (see :meth:`torch.set_default_tensor_type()`). device will be the CPU
+            Default: ``None``. If None, uses the current device for the default tensor
+            type (see :meth:`torch.set_default_tensor_type()`). Device will be the CPU
             for CPU tensor types and the current CUDA device for CUDA tensor types.
 
     Returns:
         LieTensor: a :obj:`so3_type` LieTensor
 
-    Given the expected distance :obj:`sigma`, we first calculte the standard deviation of the 
-    individual components of the tangent perturbation :math:`\sigma_{\mathrm{r}}` as:
-
-    .. math::
-        \sigma_{\mathrm{r}} = \frac{2*\sigma}{\sqrt{3}}.
-
-    The factor 2 is due to the way we define distance (see also `Matt Mason's lecture on 
-    quaternions <http://www.cs.cmu.edu/afs/cs/academic/class/16741-s07/www/lectures/Lecture8.pdf>`_)
-    The :math:`1/\sqrt{3}` factor is necessary because the distribution in the tangent space is 
-    a 3-dimensional Gaussian, so that the *length* of a tangent vector needs to be scaled by
-    :math:`1/\sqrt{3}`.
+    Note:
+        The uniform distributed points :math:`[\delta_x', \delta_y', \delta_z']` on a sphere
+        are sampled using:
 
-    Then the output can be written as:
+        .. math::
+            \delta_x' = \frac{x_0}{d}, \delta_y' = \frac{y_0}{d}, \delta_z' = \frac{z_0}{d},
+        
+        where
 
-    .. math::
-        \mathrm{out}_i = \mathcal{N}(\mathbf{0}_{3\times 1}, \mathbf{\sigma_{\mathrm{r}}}_{3\times 1}),
+        .. math::
+            x_0, y_0, z_0 \sim \mathcal{N}(0, \sigma),
 
-    where :math:`\mathcal{N}` denotes Gaussian distribution.
+        .. math::
+            d = \sqrt{(x^2_0+y^2_0+z^2_0)}.
 
-    Note:
-        The detailed explanation of the above implementation can be found in the 
-        `OMPL code <https://ompl.kavrakilab.org/SO3StateSpace_8cpp_source.html>`_, line 119.
+        Note that the point :math:`[x_0, y_0, z_0]` follows 3-D Gaussian distribution, 
+        which is centrosymmetry about :math:`(0, 0, 0)`. Thus the normalized point 
+        :math:`[\delta_x', \delta_y', \delta_z']` is both centrosymmetry and satisfies 
+        :math:`\delta_x'^2+\delta_y'^2+\delta_z'^2=1`. Therefore we have 
+        :math:`[\delta_x', \delta_y', \delta_z'] \sim \mathcal{U}_{\mathrm{s}}`.
+        The point :math:`[\delta_x', \delta_y', \delta_z']` can also represent a rotation axis, 
+        which is evenly distributed.
 
     Example:
         >>> pp.randn_so3(2, sigma=0.1, requires_grad=True, dtype=torch.float64)
         so3Type LieTensor:
-        tensor([[-0.0427, -0.0149,  0.0948],
-                [ 0.0607,  0.0473,  0.0703]], dtype=torch.float64, requires_grad=True)
+        tensor([[-0.0344,  0.0177,  0.0252],
+                [-0.0040,  0.0032,  0.0149]], dtype=torch.float64, requires_grad=True)
```

Comment:
consider add another example:
```
        >>> pp.randn_so3()
        so3Type LieTensor:
        LieTensor([ 0.4811, -0.1487, -0.5949])  # Shape (3)
        >>> pp.randn_so3(1)
        so3Type LieTensor:
        LieTensor([[0.0235, 0.0334, 1.2601]])   # Shape (1, 3)
```
So that users will know how to generate a single `so3` lietensor.
since `pp.randn_so3(1)` will generate a lietensor with shape `(1, 3)`